source/utils/StackTraceWinMSVC.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "utils/StackTracePrint.h"
+
+#include <SFML/System.hpp>
+
+#define PACKAGE "OpenDungeon"
```

Comment:
It is needed for MinGW bfd.h (it requires PACKAGE to be defined or it won't compile). Here, it is just a wrong copy/paste. I will remove it (and fix the typo in MinGW file)